pkgs/development/libraries/libfprint/default.nix

```diff
@@ -1,16 +1,28 @@
-{ stdenv, fetchurl, pkgconfig, meson, ninja, libusb, pixman, glib, nss, gtk3
-, coreutils, gtk-doc, docbook_xsl, docbook_xml_dtd_43 }:
+{ thinkpad ? false, stdenv, fetchFromGitHub, fetchurl, pkgconfig, meson, ninja, libusb, pixman, glib, nss, gtk3
+, coreutils, gtk-doc, docbook_xsl, docbook_xml_dtd_43, openssl ? null }:
+
+assert thinkpad -> openssl != null;
 
 stdenv.mkDerivation rec {
-  name = "libfprint-${version}";
+  name = "libfprint-${version}" + stdenv.lib.optionalString thinkpad "-thinkpad";
```

Comment:
Please use `pname` and reuse it anywhere there's a reference to `libfprint` (not in meta though)

When attributes `pname` and `version` are present they get concatenated with `-` to make `name`.

Reply:
fixed